.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: clojure
 lein: lein2
 jdk:
-- oraclejdk7
-- openjdk7
+  - oraclejdk7
+  - openjdk7
+script: ./ext/travisci/test.sh
+notifications:
+  email: false
```

Comment:
I decided not to because it's opt-in now.  Travis says:

```
This job is running on container-based infrastructure, which does not allow use of 'sudo', setuid and setguid executables.
If you require sudo, add 'sudo: required' to your .travis.yml
See http://docs.travis-ci.com/user/workers/container-based-infrastructure/ for details.
```